common/app/views/fragments/atoms/quiz.scala.html

```diff
@@ -102,6 +103,26 @@
     }
 }
 
+@renderHeader(maybeResults: Option[QuizResults]) = {
```

Comment:
same with name here actually (which I'm writing after the classname one – sorry if that's confusing!)